test/tests/transfers/multiapp_copy_transfer/array_variable_transfer/master.i

```diff
@@ -0,0 +1,49 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+[]
+
+[Variables]
+  [v]
+    order = FIRST
+    family = LAGRANGE
+    components = 2
+  []
+[]
+
+[Problem]
+  type = FEProblem
+  solve = false
+[]
+
+[Executioner]
+  type = Transient
+  num_steps = 1
+  solve_type = 'PJFNK'
+  petsc_options_iname = '-pc_type -pc_hypre_type'
+  petsc_options_value = 'hypre boomeramg'
+[]
+
+[MultiApps]
+  [./sub]
+    type = FullSolveMultiApp
+    input_files = sub.i
+    execute_on = initial
+  [../]
+[]
+
+[Transfers]
+  [./from_sub]
+    type = MultiAppCopyTransfer
+    direction = from_multiapp
+    source_variable = u
```

Comment:
So array and "multiple variables" aren't the same thing. This test is testing the former, but the title of this PR is about "multiple variables" Is that just incorrect?

Reply:
Yes this PR addresses both "multiple variables" and transfers of array variables. I will change the description to be more accurate.